lib/screens/v2/widget_instance/elevator_closures.ex

```diff
@@ -1,22 +1,60 @@
 defmodule Screens.V2.WidgetInstance.ElevatorClosures do
   @moduledoc false
 
-  alias Screens.Alerts.Alert
-  alias ScreensConfig.Screen
-  alias ScreensConfig.V2.Elevator
+  alias Screens.Stops.Stop
 
-  defstruct screen: nil,
-            alerts: nil
+  defstruct ~w[id in_station_alerts other_stations_with_alerts]a
 
   @type t :: %__MODULE__{
-          screen: Screen.t(),
-          alerts: list(Alert.t())
+          id: String.t(),
+          in_station_alerts: list(__MODULE__.Alert.t()),
+          other_stations_with_alerts: list(__MODULE__.Station.t())
         }
 
-  def serialize(%__MODULE__{screen: %Screen{app_params: %Elevator{elevator_id: id}}}) do
-    %{id: id, in_station_alerts: [], outside_alerts: []}
+  defmodule Station do
+    @moduledoc false
+
+    alias Screens.V2.WidgetInstance.ElevatorClosures.Alert
+
+    @derive Jason.Encoder
+
+    defstruct ~w[id name routes alerts]a
+
+    @type t :: %__MODULE__{
+            id: Stop.id(),
+            name: String.t(),
+            routes: list(String.t()),
```

Comment:
Should this be `list(Route.id())`? Might also be nice to distinguish between "routes" (the whole entity) and "route IDs", by naming this field `route_ids`.

Reply:
Ah, great, then the type can also be `Route.icon()`. (though I think we tend to call this a "route _pill_" in the frontend... inconsistencies)